website/docs/test.mdx

```diff
@@ -0,0 +1,95 @@
+---
+sidebar_label: Test Protosaurus
+---
+
+# Test Protosaurus
+
+import Description from "@theme/Description";
+import Definition from "@theme/Definition";
+import DefinitionHeader from "@theme/DefinitionHeader";
+import RpcDefinition from "@theme/RpcDefinition";
+import RpcDefinitionHeader from "@theme/RpcDefinitionHeader";
+import RpcDefinitionDescription from "@theme/RpcDefinitionDescription";
+import RpcMethodText from "@theme/RpcMethodText";
+import ProtosaurusImage from "@theme/ProtosaurusImage";
+
+<ProtosaurusImage />
+
+## Messages
+
+<Definition>
+
+<DefinitionHeader name="message">
+
+### Booking
+
+</DefinitionHeader>
+
+Represents the booking of a vehicle.
+
+Vehicles are some cool shit. But drive carefully.
+
+```protosaurus--booking.v1.Booking
```

Comment:
Yes, as long as it can be identified with the correct namespace (`booking.v1.Booking`) and `protosaurus` language, the rehype plugin will pick it up.

Reply:
I forgot to ask: are there concerns regarding this? It's still very minimal right now (e.g. we can only link to a particular package and not mixed), but I think we can expand it later, so that, say, we can create a code block with fields that can link to multiple packages.

Reply:
Sounds good.